packages/filecoin-api/test/context/mocks.js

```diff
@@ -0,0 +1,54 @@
+import * as Server from '@ucanto/server'
+
+const notImplemented = () => {
+  throw new Server.Failure('not implemented')
+}
+
+/**
+ * @param {Partial<{
+ * filecoin: Partial<import('../../src/types').StorefrontService['filecoin']>
+ * piece: Partial<import('../../src/types').AggregatorService['piece']>
+ * aggregate: Partial<import('../../src/types').DealerService['aggregate']>
+ * deal: Partial<import('../../src/types').DealTrackerService['deal']>
```

Comment:
If we can turn off the no indent rule that would be helpful ;)

Reply:
it is prettier who is putthing this. could not disable it without affecting everywhere, which would not be desirable